client/js/widgets/widget.js

```diff
@@ -2467,6 +2479,7 @@ export class Widget extends StateManaged {
     state.owner = null;
     state.linkedToSeat = null;
     state.onlyVisibleForSeat = null;
+    state.onlyUsableBySeat = null;
```

Comment:
Probably not necessary to remove for a readonly copy because it's also a readonly thingy. But doesn't hurt either.

Reply:
I put it there because it seemed to fit with onlyVisibleForSeat. But testing shows it is not needed.  Removed.